### ISSUE TITLE
calendar postscript.sql failed

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#91]
```
What steps will reproduce the problem?
1. psql -f plugins/calendar/SQL/postgresql.sql

In psql (PostgreSQL) 8.3.9 we need 
quotes in line 8 around the word end 
and a point "," at the end of line 23


```

Original issue reported on code.google.com by `conradsh...@googlemail.com` on 12 Aug 2010 at 6:21

**Comment by GoogleCodeExporter** [comment]
```
Please attach corrected script. I do not use PostgeSQL. The included script is 
a contribution of a plugin user.
```

Original comment by `rol...@liebl.ath.cx` on 12 Aug 2010 at 8:03
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
greetings ...
```

Original comment by `conradsh...@googlemail.com` on 12 Aug 2010 at 10:13

Attachments:
- [postgres.sql](https://storage.googleapis.com/google-code-attachments/myroundcube/issue-91/comment-2/postgres.sql)

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 12 Aug 2010 at 2:21

**Comment by GoogleCodeExporter** [comment]
```
Thanks for contribution.
```

Original comment by `rol...@liebl.ath.cx` on 13 Aug 2010 at 8:36

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3878.1.
```

Original comment by `rol...@liebl.ath.cx` on 14 Aug 2010 at 4:54
- Changed state: **Fixed**
